lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -60,6 +60,14 @@ class PlatformViewManager {
     return _contents.containsKey(viewId);
   }
 
+  /// Returns the HTML element created by a registered factory for [viewId].
+  ///
+  /// Throws an [AssertionError] if [viewId] hasn't been rendered before.
+  DomElement getViewById(int viewId) {
+    assert(knowsViewId(viewId), 'No view has been rendered for viewId: $viewId');
+    return _contents[viewId]!.firstElementChild!;
```

Comment:
Please, document that we return the `firstElementChild` because we cache the created view wrapped with the `<flt-slot-content>` tag (right?)

Reply:
`_contents[viewId]` is the wrapper (i.e. the slot element). The first child of the slot is the element created by the factory.

Reply:
https://github.com/flutter/engine/blob/f3f6a020595dd7f9f809bca2ebb604a42aae1cfc/lib/web_ui/lib/src/engine/platform_views/content_manager.dart#L133

Reply:
Oh I think this is exactly what you are saying. Sorry I misread your comment.

Sounds good, I'll document why I'm returning `firstElementChild`.